docs/docsite/rst/reference_appendices/general_precedence.rst

```diff
@@ -0,0 +1,131 @@
+.. _general_precedence_rules:
+
+Controlling how Ansible behaves: precedence rules
+=================================================
+
+To give you maximum flexibility in managing your environments, Ansible supports many ways to control how Ansible behaves: how it connects to managed nodes, how it works once it has connected.
+If you use Ansible to manage a large number of servers, network devices, and cloud resources, you may define Ansible behavior in several different places and pass that information to Ansible in several different ways.
+This flexibility is convenient, but it can backfire if you do not understand the precedence rules.
+
+These precedence rules apply to any setting that can be defined in multiple ways (by configuration settings, command-line options, playbook keywords, variables). For example, defining connection information, selecting a temporary directory, and choosing the correct python/ruby/powershell/etc interpreter to invoke for a module using settings like ANSIBLE_USER, DEFAULT_LOCAL_TMP, or ANSIBLE_PYTHON_INTERPRETER all follow these precedence rules.
+
+.. contents::
+   :local:
+
+Precedence categories
+---------------------
+
+Ansible supports four sources for controlling its behavior. In order of precedence from lowest (most easily overridden) to highest (overrides all others), the categories are:
+
+   Configuration settings
+   Command-line options
+   Playbook keywords
+   Variables
+
```

Comment:
Consider flipping this on its head.  When I think "highest", I naturally expect to see it at the top of this bullet list. That might jus be my mental quirk so take w/ a grain o salt.

Reply:
My instinct is the same, but I vote we save that for a future PR. If we flip this list, we also need to flip the [precedence-among-variables list](https://docs.ansible.com/ansible/devel/user_guide/playbooks_variables.html#ansible-variable-precedence), and this PR has enough scope creep already.